src/main/kotlin/step2/Operator.kt

```diff
@@ -0,0 +1,16 @@
+package step2
+
+enum class Operator(
+    val char: String
+) {
+    PLUS("+"),
+    MINUS("-"),
+    MULTIPLY("*"),
+    DIVISION("/"),
+    ;
+
+    companion object {
+        fun of(char: String): Operator = Operator.values().find { it.char == char }
+            ?: throw IllegalArgumentException(ErrorMessage.INVALID_OPERATOR)
```

Comment:
엘비스 연산자를 잘 활용해주셨어요 👍 
만약 
```suggestion
        fun of(char: String): Operator = values().find { it.char == char }
            ?: throw IllegalArgumentException(ErrorMessage.INVALID_OPERATOR)
```
nullable한 결과를 반환하는 `find` + 엘비스 연산자(`?:`) 활용 잘 해주셨어요 👍 항상 결과를 반환하는 `first`도 한 번쯤 체크해보고 넘어갈만 한 것 같습니다.
그리고 `Operator`를 명시해주지 않아도 될 것 같아요!